services/api/src/api/routes/croissant.py

```diff
@@ -231,6 +259,14 @@ def _get_full_jsonld_parameter(request: Request) -> bool:
     return True
 
 
+def _get_use_croissant_1_0(request: Request) -> bool:
+    """Whether the output follows Croissant 1.0 or 0.8. Defaults to True."""
+    use_croissant_1_0 = get_request_parameter(request, "use_croissant_1_0", default="true")
```

Comment:
Not sure about the snake_case for the URL parameter. From their documentation it looks like HF favors short words. Maybe `?v1=true` or `?isV1=true`?

Reply:
@severo The idea would be to have a boolean flag to first control that the migration to 1.0 is ok (e.g., using [the crawler](https://github.com/mlcommons/croissant/blob/main/health/crawler/spiders/huggingface.py)) and then do the switch. WDYT?

Reply:
Changed to `isV1`, done.